composer.json

```diff
@@ -30,11 +30,13 @@
         "drupal/entity_clone": "^1.0@beta",
         "drupal/entityqueue": "1.x-dev",
         "drupal/fontyourface": "3.x-dev",
+        "drupal/google_api_client": "^3.0",
```

Comment:
can you remove this, please? and redo the composer install.  also don't forget to enable the module and export the core.extensions.yml file.